data-raw/testdata.R

```diff
@@ -0,0 +1,271 @@
+# Script to generate synthetic data for tests
+
+# Load required libraries
+library(tidyverse)
+library(data.table)
+library(here)
+library(lubridate)
+
+# Set seed for reproducibility
+set.seed(123)
+
+# Set number of rows to create for fake data
+number_rows <- 1000
+
+
+# Functions ---------------------------------------------------------------
+
+generate_fake_atc <- function(number_rows) {
+  codeCollection::ATCKoodit |>
+    tibble::as_tibble() |>
+    dplyr::filter(stringr::str_length(Koodi) == 7) |>
+    dplyr::pull(Koodi) |>
+    sample(number_rows, replace = TRUE)
+}
+
+generate_fake_indication <- function(number_rows) {
+  sample(1:9e8, number_rows, replace = TRUE) |>
+    stringr::str_trunc(width = 7, ellipsis = "") |>
+    stringr::str_pad(width = 7, pad = "0")
+}
```

Comment:
I moved this into a function, but I'm not sure what this is supposed to be since I can't see the description in the #4 issue.

Reply:
It's good :)

(I've written a bit about indication codes below).